DOCS: update README (#121)
django-cms/djangocms-picture: README.rst

    :target: https://www.djangoproject.com/
.. |djangocms| image:: https://img.shields.io/badge/django%20CMS-3.7%2B-blue.svg
    :target: https://www.django-cms.org/

Updating from `cmsplugin-filer <https://github.com/django-cms/cmsplugin-filer>`_
--------------------------------------------------------------------------------

> `cmsplugin-filer` was used a few times ago to create file, folder, image, link, teaser & video plugins on your DjangoCMS projects. Now the project is archived, and you might be wondering how to migrate your old instances without having to copy them manually to the new `djangocms-<file|picture|link|...>` plugins.

Reviewer: ```suggestion
Historically, `cmsplugin-filer` was used to create file, folder, image, link, teaser & video plugins. Now `cmsplugin-filer` is archived, you can still migrate your old instances without having to copy them manually to the new `djangocms-<file|picture|link|...>` plugins.
```

Author: I fixed the content, thanks!

> Don't be afraid, we have you covered! Here's a management command that you can use to get started with this tasks:

Reviewer: ```suggestion
There's a third-party management command that supports your migration:
```

`migrate_cmsplugin_filer.py <https://gist.github.com/corentinbettiol/84a6ea7e4d047fc01861b0af15fd60f0>`_

> This management command is only a starting point. It *may* work out of the box for some people, but we encourage you to read the code, understand what it does, and try it on a development environment before running it on your production server. The management command is only configured to transfer your `cmsplugin_link`, `cmsplugin_file`, `cmsplugin_folder` and `cmsplugin_image` plugins to modern `djangocms_*` plugins. If you need to transfer other `cmsplugin_*` plugins, you'll have to write your own code.

Reviewer: ```suggestion
This management command is only a starting point. It *has* worked out of the box for some people, but we encourage you to read the code, understand what it does, and test it on a development environment before running it on your production server. 

The management command is configured to transfer your `cmsplugin_link`, `cmsplugin_file`, `cmsplugin_folder` and `cmsplugin_image` plugins to modern `djangocms_*` plugins. If you need to transfer other `cmsplugin_*` plugins, you'll have to write your own code.
```
